charts/sentry/templates/sentry/metrics/billing/deployment-sentry-billing-metrics-consumer.yaml

```diff
@@ -91,14 +91,6 @@ spec:
           - "/tmp/health.txt"
           {{- end }}
           - "--"
```

Comment:
That line must be last in whole scope of cmd parameters
otherwise process throws error